docs/registry/create/_index.md

```diff
@@ -670,9 +670,14 @@ Make sure to set up a Python virtual environment in the directory your module is
    Add additional dependencies as needed.
    See the [pip `requirements.txt` file documentation](https://pip.pypa.io/en/stable/reference/requirements-file-format/) for more information.
 
-2. Add a shell script that creates a new virtual environment, installs the dependencies listed in `requirements.txt`, and runs the module entry point file `main.py`:
+From here, you have three options for how to bundle your module: using `venv`, `nuitka`, or `pyinstaller`.
```

Comment:
```suggestion
1. From here, you have three options for how to bundle your module: using `venv`, `nuitka`, or `pyinstaller`.
   Select a tab below and follow the instructions for your chosen option:
```

I'd recommend retaining this as an explicit step, since the users must choose which, and then click on a tab. Also, making this an explicit step, allows us to remove the `1.` step below, within the `venv` tab, and avoid having steps that cross tab boundaries (which reset back to 1 within a tab, for example).